about.html

```diff
@@ -0,0 +1,155 @@
+                        <p>"Adorable Forex Trading"</p>
+                        <h1>FIRST YOU LEARN, <br> THEN YOU TAKE OUT THE 'L'</h1>
+                    </div>
+                
+                    
+                        <P class="aboutPara1">Adorable Forex Trading is one of the leading forex 
+                            and commodities broker focused on providing 
+                            top-notch financial trading solutions, 
+                            facilities, and efficient customer support 
+                            services to our wide range of clients.
+                        </P>                        
+                    <div class="aboutPara2">
+                        <p>Please contact us per Email for any further 
+                            questions about Adorable Forex Trading!
+                        </p>
+                        <p> <a href=""> adorableforex@gmail.com</a></p>
+                    </div>
+            </div>
+        </section>   
+    </section>
+
+    <div class="footerContainerLogo">
+        <section class="aboutLogo aboutLogo2">
+            <p class="aboutLogo-P1">First You Learn, Then You Take Out The 'L'</p>
+            
+            <hr>
+            <p class="aboutLogo-P2">The logo of Adorable Forex Trading was decided 
+                through the logo competition from 8.June to 7.JUly.
+            </p>
+    
+            <div class="logoImg logoImg2">
+                <img class="aboutL" src="./images/myLogo.jpg" alt="Logo">
+                <p>Adorable Forex Trading</p>
+            </div>
+        </section>
+        
+        <div>
+            <hr class="footerHr">
+            <div class="footerPara1">
+                <p class="footerParaP">
+                    See the past AFT Achievements
+                </p>
+                <hr>
+                <p class="footerPar">
+                    Take a look at the last two CC 
+                    Gobal Summits which took place in Buenos 
+                    Aires and in Warsaw.
+                </p>
+            </div>
+
+            <div class="forexClass forex">
+                <div class="forexClass1">
+                    <h5>
+                        2021
+                    </h5>
+                    <p>Forex Analysis</p>
+                </div>
+
+                <div class="forexClass2">
+                    <h5>
+                        2019
+                    </h5>
+                    <p>Signals Daily Posting</p>
+                </div>
+            </div>
+
+        </div>
+    </div>
+
+    <footer class="footer">   
+        <div class="footerPartner" id="footerPartner">
+            <p>Partner</p>
+            <hr>
+            <div class="partnerLogo">
+                <p>Mozilla</p>
+                <p>Google</p>
+                <p>Naver</p>
+                <p>daumkakao</p>
+                <p>Google</p>
+                <p>Naver</p>
+            </div>
```

Comment:
- [ ] The footer with the dark background on the about page is missing. Kindly add that, please.
![image](https://user-images.githubusercontent.com/94551484/218193671-343b2db5-60e0-412a-b992-7e882c8bcfd2.png)